lms/static/sass/_header.scss

```diff
@@ -0,0 +1,365 @@
+          display: none;
+          border: 1px solid theme-color("secondary");
+          position: absolute;
+          background-color: $white;
+          color: theme-color("secondary");
+          right: 30px;
+          top: 50px;
+          z-index: 10;
+
+          .dropdown-item {
+            padding: $baseline*0.4 $baseline;
+            cursor: pointer;
+            font-weight: 600;
+
+            &:hover {
+              color: theme-color("dark");
+              background-color: theme-color("light");
+            }
+
+            a:focus{
+              outline: none;
+              color: theme-color("primary");
+            }
+
+            &:not(:last-child) {
+              border-bottom: 1px solid theme-color("light");
+            }
+          }
+        }
+      }
+    }
+    .hamburger-menu {
+      display: none;
+    }
+  }
+
+  // Responsive styling for mobile
+  @include media-breakpoint-down(sm) {
+    // Styles for the transition from open to collapsed
+    .nav-links {
+      display: none;
+      -webkit-transition: height 2.0s;
+      -moz-transition: height 2.0s;
+      transition: height 2.0s;
+
+      &.nav-links-expanded {
+        display: block;
+      }
+
+      .nav-item-not-collapsible { // Hide any items that we do not want to collapse
+        display: none;
+      }
+
+      .nav-item-not-styled { // Remove specific styles from any items that we do not want to style on mobile
+        margin-top: 0;
+      }
+    }
+
+    // Display and style all mobie menu elements
+    .nav-item,
+    .dropdown-nav-item {
+      display: inline-block;
+      width: 100%;
+      border: none;
+      padding: $baseline/5 0 0;
+      margin: $baseline*0.75 $baseline 0;
+      color: theme-color("primary");
+      border: none;
+      text-decoration: none;
+      background-color: transparent;
+      font-weight: normal;
+      font-size: 20px;
+
+      a {
+        text-decoration: none;
+
+        .active, // Indicates if we are currently on one of the pages
+        &:hover{
+          color: theme-color("dark");
+        }
+      }
+
+      &:hover {
+        color: theme-color("dark");
+        font-weight: 600;
+      }
+    }
+
+    // Custom focus behavior for tabbing through elements (links, dropdown links, and buttons)
+    .nav-item a:focus,
+    .dropdown-nav-item a:focus,
+    a.nav-item:focus {
+      outline: none;
+      color: theme-color("dark");
+      font-weight: 600;
+    }
+
+
+    // Custom styling to ensure dropdown elements match other nav elements in collapsed mode
+    .dropdown-user-menu {
+      display:inline !important;
+
+      .dropdown-nav-item {
+        @include margin-left(0);
+      }
+
+      a:first-child{
+        .dropdown-nav-item {
+          margin-top: 0;
+        }
+      }
+    }
+
+    // Display the menu icon and allow for transition to an X on click
+    .hamburger-menu {
+      @include right(22px);
+      position: absolute;
+      top: 24px;
```

Comment:
There are a lot of hard-coded "magic" values in this file. Consider making more of them into semantic variables so that it is clearer what they are, and so that a theme could override theme if necessary.

Reply:
Since the hamburger menu is so custom, I'm going to leave these values as is, but moving a bunch of the other variables into the bootstrap variables file so that themes can easily override them